### PR TITLE
Application initial setup.

### PR DESCRIPTION
**Pull request by Merdoth**
#### What does this PR do?
- Initial application setup
- Add webpack file and config settings for both dev and prod application.
- Modify eslintrc rules.

#### Description of Task to be completed?
- Setup the general client side and server side structure.
- Included basic files in structure to enable the team knows the exact file convention to follow.

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- story type: Chore
- story title: Application initial setup

#### Screenshots (if appropriate)
N/A

#### Question
N/A

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,900|Oswald:400,600,700,900|Open+Sans:800" rel="stylesheet">
+
+</head>
+
+<body>
+  <div id="root"></div>
+    <script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.1/js/materialize.min.js"></script>
+  <script src="/bundle.js"></script>
+  <script src="https://use.fontawesome.com/e7318f820e.js"></script>
```
Comment (enyeInc) [inline client/index.html]: minor nit: name sure `script` tags are lined up

File: client/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Kurale" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,900|Oswald:400,600,700,900|Open+Sans:800" rel="stylesheet">
+
+</head>
+
+<body>
+  <div id="root"></div>
+    <script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
```
Comment (enyeInc) [inline client/index.html]: Do not use `JQuery`

File: client/index.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.1/js/materialize.min.js"></script>
```
Comment (enyeInc) [inline client/index.html]: Use the material [npm module](https://www.npmjs.com/package/@material-ui/core), not the cdn

File: client/js/app/App.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import history from '../utils/history';
+
+const App = () => (
+  <Router history={history}>
+    <Switch>
+      <Route
+        exact
+        path="/"
+        name="landing"
+        // component={(Home)}
```
Comment (enyeInc) [inline client/js/app/App.jsx]: remove unused code

File: client/index.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+<head>
+  <meta charset="UTF-8">
+  <title>Life Aid</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Kurale" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,900|Oswald:400,600,700,900|Open+Sans:800" rel="stylesheet">
+
```
Comment (enyeInc) [inline client/index.html:14]: delete empty line

**Comment by enyeInc** [comment]
@Merdoth make the minor fixes and this should be good to merge down

**Review by iammiracle — APPROVED** [review]
Looking okay.

File: client/index.html (hunk 1 above)
Comment (Merdoth) [reply client/index.html]: Done

File: client/index.html (hunk 2 above)
Comment (Merdoth) [reply client/index.html]: Removed

File: client/index.html (hunk 3 above)
Comment (Merdoth) [reply client/index.html]: Removed

File: client/js/app/App.jsx (hunk 4 above)
Comment (Merdoth) [reply client/js/app/App.jsx]: Uncommented it

File: client/index.html (hunk 5 above)
Comment (Merdoth) [reply client/index.html:14]: Done

**Review by codemutatio — APPROVED** [review]
Looks good

**Comment by enyeInc** [comment]
@Merdoth this looks ready to be merged

**Pull request merged**